### ISSUE TITLE
Finger change elision marks

### ISSUE THREAD

**Issue by craigsapp** [music-encoding/music-encoding#560]
How should finger change markings in the following examples be encoded in MEI:

First where there is an elision mark below the numbers (but not necessarily a gap between the numbers as would be found in lyrics, since they are single-digits):

![fing-switch3](https://user-images.githubusercontent.com/29225574/45431925-e3f44e80-b6a8-11e8-8583-a5e744d2896b.png)

And a similar case, but the elision is above the numbers:

![fing-switch2](https://user-images.githubusercontent.com/29225574/45432193-8c0a1780-b6a9-11e8-84a2-e00152381f88.png)

Perhaps the elision direction syntax is similar to fermatas (the arc of the elision moves away from the staff).

Related to issue https://github.com/humdrum-tools/verovio-humdrum-viewer/issues/147

**Comment by pe-ro** [comment]
> Perhaps the elision direction syntax is similar to fermatas (the arc of the elision moves away from the staff).

This was always my assumption.  At present, there's no way to control for those cases where this doesn't apply, but we can add an attribute in att.fingGrp.vis to account for them.

**Comment by bwbohl** [comment]
Is there a need of controlling elision direction otherwise I would consider to close this issue?

**Comment by bwbohl** [comment]
ping

**Comment by pe-ro** [comment]
I would recommend closing this issue for now.  It can be re-opened if there's a pressing need for indicating fingering elision direction.

**Comment by rettinghaus** [comment]
Agreed. I have some ideas about this, but it will be a bigger construction site. Closing for now.

**Comment by bwbohl** [comment]
as closing marks issues as completed and moves them "out of sight" I reopened this and Moved it to the MEI 5.x Milestone backlog instead.
